crates/task-impls/src/vid.rs

```diff
@@ -220,12 +217,24 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
             }
 
             HotShotEvent::TransactionsSequenced(encoded_transactions, metadata, view_number) => {
+                // get quorum committee for dispersal
+                let num_quorum_committee = self.membership.get_committee(view_number).len();
+
                 // TODO <https://github.com/EspressoSystems/HotShot/issues/1686>
-                let srs = test_srs(NUM_STORAGE_NODES);
-                // TODO We are using constant numbers for now, but they will change as the quorum size
-                // changes.
-                // TODO <https://github.com/EspressoSystems/HotShot/issues/1693>
-                let vid = VidScheme::new(NUM_CHUNKS, NUM_STORAGE_NODES, &srs).unwrap();
+                let srs = test_srs(num_quorum_committee);
+
+                // calculate the last power of two
+                // TODO change after https://github.com/EspressoSystems/jellyfish/issues/339
```

Comment:
Could we add a HotShot issue for this TODO, to make sure we won't forget to fix it later?